android/expoview/src/main/java/host/exp/exponent/notifications/schedulers/HashMapSerializer.java

```diff
@@ -0,0 +1,73 @@
+package host.exp.exponent.notifications.schedulers;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+
+public class HashMapSerializer {
```

Comment:
Couldn't we use `Arguments.toBundle` to convert `ReadableMap` into something serializable (`Bundle`) instead of writing our own serializer?

Reply:
To be changed when moving to unimodules — `ExponentNotificationManager` expects `HashMap`s.